<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2600</width>
    <height>1700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>401</width>
      <height>1306</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0.370787 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
border-radius:30px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="wareForm">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>690</y>
       <width>381</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:40;
background-color: rgb(255, 255, 255);

</string>
     </property>
     <property name="text">
      <string>库存报表</string>
     </property>
    </widget>
    <widget class="QPushButton" name="suggestion">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>990</y>
       <width>381</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>28</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:40;
background-color: rgb(255, 255, 255);

</string>
     </property>
     <property name="text">
      <string>建议</string>
     </property>
    </widget>
    <widget class="QPushButton" name="profit">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>390</y>
       <width>381</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>28</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:40;
background-color: rgb(255, 255, 255);

</string>
     </property>
     <property name="text">
      <string>收益</string>
     </property>
    </widget>
    <widget class="QPushButton" name="wareHouse">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>381</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>28</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:40;
background-color: rgb(85, 170, 255);


</string>
     </property>
     <property name="text">
      <string>仓库</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="getIn">
    <property name="geometry">
     <rect>
      <x>573</x>
      <y>33</y>
      <width>376</width>
      <height>228</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>48</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(12, 106, 239);
border-radius:36px;</string>
    </property>
    <property name="text">
     <string>入库</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getOut">
    <property name="geometry">
     <rect>
      <x>1520</x>
      <y>33</y>
      <width>376</width>
      <height>228</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>48</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(12, 106, 239);
border-radius:36px;</string>
    </property>
    <property name="text">
     <string>出库</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>2120</x>
      <y>360</y>
      <width>454</width>
      <height>1163</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));
border-radius:30px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string> 仓库名：</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>100</y>
       <width>291</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));
border-bottom-color: rgb(85, 0, 0);
border:30px;</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>330</y>
      <width>1341</width>
      <height>1231</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
